index.d.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference types="node"/>
+
+/**
+ * Read a chunk from a file asyncronously.
+ *
+ * @param filePath - The path to the file.
+ * @param startingPosition - Position to start reading.
```

Comment:
If you improve parameter names or descriptions, would be great if you could update that in the readme too, so they stay in sync.